### ISSUE TITLE
Broken 'nvm use' command on v1.1.9

### ISSUE THREAD

**Issue by guilhermejcgois** [coreybutler/nvm-windows#870]
**Issue:**

Can't use installed versions of node through `nvm use` command. I have re-installed nvm-windows 1.1.9 using scoop but the problem persist. Downgrade to 1.1.8 works as expected.

**How To Reproduce:**

1. scoop install nvm@1.1.9
2. nvm install 19
3. nvm use 19
4. See error `node v19.0.0 is not installed.`

**Expected Behavior:**

I can change the current installed node version.

**Comment by hyunbinseo** [comment]
Personally seeing a `vv18.12.0 (64-bit)` message with two `v`s.

```
PS C:\Windows\System32> nvm use 18
node vv18.12.0 (64-bit) is not installed or cannot be found.
```

Providing the full version number works in nvm version 1.1.9.

```
PS C:\Windows\System32> nvm use 18.12.0
Now using node v18.12.0 (64-bit)
```

**Comment by acambule** [comment]
I experience the same problem on v. 1.1.9,
and even if nvm tells you it works, the proof show the opposite:

![image](https://user-images.githubusercontent.com/22659382/198197057-ade57054-30f1-4f53-bcb3-0e01ef885f7b.png)

**Comment by dsl101** [comment]
Same as @hyunbinseo for me—short version broken, long version works. Not seeing the incorrect version reported as @acambule is, and confirmed working with `node --version`:

![image](https://user-images.githubusercontent.com/900542/198314701-d4b2669e-5570-44b1-bd30-0029db6a9eb8.png)

**Comment by DarkMatterMatt** [comment]
The fix has been merged in #709, but the next release date is unknown. Workaround is to explicitly use the full version (e.g. `nvm use 18.12.0` instead of `nvm use 18`).

For more info look at the issues that this is a duplicate of: [github.com/coreybutler/nvm-windows/issues?q="is not installed or cannot be found"](https://github.com/coreybutler/nvm-windows/issues?q=%22is+not+installed+or+cannot+be+found%22)

**Comment by dsl101** [comment]
Thanks—that fix was originally written nearly a year ago, so no wonder there are duplicate issues... Fingers crossed the release will be soon :).

**Comment by coreybutler** [comment]
Duplicate of #708. Resolved in NVM4W 1.1.10.
